tests/main/dirs-not-shared-with-host/task.yaml

```diff
@@ -27,7 +31,15 @@ execute: |
     else
         core_inode=$(stat -c '%i' "$SNAP_MOUNT_DIR/ubuntu-core/current/$DIRECTORY")
     fi
-    effective_inode=$(test-snapd-sh.sh -c "stat -c '%i' $DIRECTORY")
-    echo "The inode number as seen from a confined snap should be that of the $DIRECTORY from the core snap"
     [ "$host_inode" != "$core_inode" ]
-    [ "$effective_inode" = "$core_inode" ]
+
+    effective_inode=$(test-snapd-sh.sh -c "stat -c '%i' $DIRECTORY")
```

Comment:
should we instead change this test to use a different directory than ssl and make a new test that is dedicated to /etc/ssl with a clearer name? it seems a better path forward to cover what is going on

Reply:
I think this makes sense, I add it